onboarding_project/squad_pantry_app/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from datetime import datetime
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator
+
+# Create your models here.
+
+
+class Dish(models.Model):
+    dish_name = models.CharField(max_length=256, unique=True)
+    DISH_TYPE = (
+        ('NV', 'Non-Vegetarian'),
+        ('V', 'Vegetarian'),
+        ('E', 'Contains Egg'),
+    )
+    dish_type = models.CharField(max_length=2, choices=DISH_TYPE, default=DISH_TYPE[0][0])
+    is_available = models.BooleanField(default=False, blank=False)
+    estimated_preparation_time_in_minutes = models.IntegerField(validators=[MinValueValidator(1)])
+
+
+class Order(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    STATUS = (
+        ('OP', 'Order Placed'),
+        ('REJ', 'Rejected'),
+        ('ACC', 'Accepted'),
+        ('CAN', 'Cancelled'),
+        ('PROC', 'Processing'),
+        ('DEL', 'Delivered')
+    )
+    status = models.CharField(max_length=4, choices=STATUS, default=STATUS[0][0])
+    scheduled_time = models.DateTimeField(blank=True, null=True)
+    order_placed_at = models.DateTimeField(default=datetime.now())
```

Comment:
Is this the recommended way of having callables as a default value? Read about [auto_now](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.DateField.auto_now) and [auto_now_add](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.DateField.auto_now_add) and see if we can use them here

Reply:
Used `auto_now_add` before this. But it doesn't show in the django-admin. 
If that's okay and we don't need to show it in the UI and just use it at the backend I'll use it.

Reply:
Do we need to show `order_placed_at` in the UI? Why do you think we should/should not let the user edit this field from the UI?

Reply:
No need to show it to user.

Reply:
Okay. But why though? 😋